python/py-pydantic/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pydantic
+version             1.5.1
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38
+
+maintainers         {@jandemter demter.de:jan} openmaintainer
+
+description         Data validation and settings management using Python type hinting
+long_description    Fast and extensible, pydantic plays nicely \
+                    with your linters/IDE/brain. Define how data \
+                    should be in pure, canonical Python 3.6+\; \
+                    validate it with pydantic.
+
+homepage            https://github.com/samuelcolvin/pydantic
+master_sites        pypi:p/pydantic
+
+distname            pydantic-${version}
```

Comment:
`master_site` and `distname` are set to these values by the python PG, so please remove these two lines